pulser-core/pulser/sequence/sequence.py

```diff
@@ -76,6 +77,15 @@
 
 PROTOCOLS = Literal["min-delay", "no-delay", "wait-for-all"]
 
+_metadata = contextvars.ContextVar("metadata", default={"library": "pulser", "pulser_version": pulser.__version__})
```

Comment:
There is already a dedicated `pulser_version` field, I wonder if we want to replicate the info in the metadata